var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -144,12 +143,10 @@ def install(self, spec, prefix):
             )
             args.append("--cuda_compute_capabilities={0}".format(capabilities))
         args.append(
-            "--bazel_startup_options="
-            "--output_user_root={0}".format(self.wrapped_package_object.buildtmp)
+            "--bazel_startup_options=" "--output_user_root={0}".format(self.stage.source_path)
         )
         python(*args)
         with working_dir(self.wrapped_package_object.tmp_path):
             args = std_pip_args + ["--prefix=" + self.prefix, "."]
             pip(*args)
         remove_linked_tree(self.wrapped_package_object.tmp_path)
```

Comment:
Is it possible to get rid of this use of tempfile too?

Reply:
I'll check.